Fix #3269: Replace broken and outdated URLs in data.atmosphere module (#3496)
PecanProject/pecan: modules/data.atmosphere/README.md

## Documentation

* [Package Documentation](https://pecanproject.github.io/package-documentation/develop/data.atmosphere/index.html)
* Vignettes

## PEcAn variable names

## For the most updated list, see https://pecanproject.github.io/pecan-documentation/latest/time-standard.html#input-standards

> <!-- Previous link: https://pecanproject.github.io/pecan-documentation/latest/time-standard (404) -->

Reviewer: ```suggestion
For the most updated list, see https://pecanproject.github.io/pecan-documentation/develop/input-standards.html#input-standards
```

## Documentation

> ## * [Package Documentation](https://pecanproject.github.io/pecan//modules/data.atmosphere/inst/web/index.html)

Reviewer: Let's use this link -- it's currently 404 because of a deployment hiccup, but should be the right one to use here when it comes online: 
```suggestion
* [Package Documentation](https://pecanproject.github.io/package-documentation/develop/data.atmosphere/index.html)
```

* Vignettes

> <!-- Previous link: https://pecanproject.github.io/pecan//modules/data.atmosphere/inst/web/index.html (404) -->

Reviewer: All three of these look like the same URL to me. Am I missing something?

> <!-- Previous link: https://pecanproject.github.io/pecan//modules/data.atmosphere/inst/web/index.html (404) -->

Reviewer: ```suggestion
```
